<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSymbolSort</class>
 <widget class="QDialog" name="DialogSymbolSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二次符号分类</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>9</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>符号代码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_AllSymbolsView">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="listViewGroupMember">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>55</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>9</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所有符号代码：</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewAllSymbol">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>55</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_GroupMemberView">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>310</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>289</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>预览：</string>
   </property>
  </widget>
  <widget class="QListView" name="listViewGroup">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>160</width>
     <height>348</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>符号类型名称：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>389</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>8</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRemoveGroup">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>389</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>289</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>预览：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAddToGroup">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>75</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AllSymbolSearch">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAllSymbolSearch">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>30</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>检索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_MemberSearch">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnMemberSearch">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>检索</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
